experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+
+Next, install it on your local publish instance via Package Manager (you may need to [sign in first](http://localhost:4503/libs/granite/core/content/login.html)): [localhost:4503/crx/packmgr/index.jsp](http://localhost:4503/crx/packmgr/index.jsp)
+
+To verify that the package is installed correctly and is aware of our `DispatcherMaxAgeHeaderFilter` [OSGi config](https://github.com/adobe/aem-dispatcher-experiments/blob/enableTTL/aem-project/ui.apps/src/main/content/jcr_root/apps/dispatchertester/config/com.adobe.acs.commons.http.headers.impl.DispatcherMaxAgeHeaderFilter-paths-set-1.xml#L5) (included in this repo's `aem-project/`, which sets the `max.age` of `/content/dispatchertester/*` content to 60 seconds), issue the following request with cURL:
+
+```
+curl -H "Server-Agent: Communique-Dispatcher" -D - http://localhost:4503/content/dispatchertester/us/en.html -o /dev/null
+```
+
+Ensure the `Cache-Control: max-age=60` header is included in the response before continuing. 
+
+> Is the `Cache-Control` header missing? Ensure you've installed the latest code in `aem-project/` to your publish instance. To do so with a single command, (from the `aem-project/` dir): `mvn -PautoInstallSinglePackagePublish clean install`
+
+Next, set the `enableTTL` flag to "1" by un-commenting the following line in your dispatcher.any configuration (on macOS: `/private/etc/apache2/conf/dispatcher.any`, around line 255):
+
+```
+/enableTTL "1"
+```
+
+You will need to restart Apache for this change to take effect:
+
+```
+sudo apachectl restart
+```
+
+Clear your dispatcher cache of any existing content. On macOS:
+
+```
+sudo rm -rf /Library/WebServer/docroot/publish/*
+```
+
+Finally, navigate to regular-page.html and observe the files which are written to the cache:
+
+http://aem-publish.local:8080/content/dispatchertester/us/en/regular-page.html
+
+Result:
+
+<img src="../img/enableTTL-ttl-file.png" width="900">
+
+Note how the last modified timestamp on `regular-page.html.ttl` is actually set _in the future_ by a minute - our `max.age`. Once the current time is _after_ the last modified timestamp on a resource's .ttl file, the file will be considered stale. 
+
+## In the logs
+
+The dispatcher.log output will differ slightly when using `enableTTL` vs. .stat file-based invalidation.
+
+### With `enableTTL`
+
+With `enableTTL` set and a corresponding .ttl file next to the cached resource in question, you will see log entries that match the following format.
```

Comment:
Since this sentence ends with "match the following format." it was a bit jarring at first to go into another sentence below, not a format.  For readability I would suggest the two formats below be indented (possibly bullet items) and "match the following format:" ends with a colon.

Reply:
Should there be some instructions on how to tail those logs?  Screenshots?  I know we do that in previous dispatcher experiments so is it assumed they do all these in order, or can somebody cherry pick this one experiment?

Reply:
I added a bit on where to find the files:

> In a typical install on macOS, dispatcher.log can be found here: `/private/var/log/apache2/dispatcher.log`

Without tail someone could use the Console app which I believe is the default for .log, and it actually works quiet well.